omod/src/main/resources/config.xml

```diff
@@ -1,117 +1,123 @@
-	<author>owais.hussain</author>
-	<description>
-			${project.parent.description}
-	</description>
-
-	<activator>org.openmrs.module.commonlabtest.CommonLabTestActivator</activator>
-	
-	<!-- <updateURL>https://modules.openmrs.org/modules/download/@MODULE_ID@/update.rdf</updateURL> -->
-	<!-- /Base Module Properties -->
-	
-	<require_version>2.0.3</require_version>
-
-	<require_modules>
-		<require_module version="${wsrestVersion}">org.openmrs.module.webservices.rest</require_module>
-	</require_modules>
-	
-	<!-- Extensions -->
-	<extension>
-		<point>org.openmrs.admin.list</point>
-		<class>org.openmrs.module.commonlabtest.extension.html.AdminList</class>
-	</extension>
-
-	<extension>
-		<point>org.openmrs.patientDashboardTab</point>
-		<class>${project.parent.groupId}.${project.parent.artifactId}.extension.html.CommonLabTestExt</class>
-	</extension>
-
-	<aware_of_modules>
-		<aware_of_module>org.openmrs.module.legacyui</aware_of_module>
-	</aware_of_modules>
-	
-	
-	<!-- AOP
-	<advice>
-		<point>org.openmrs.api.FormService</point>
-		<class>@MODULE_PACKAGE@.advice.DuplicateFormAdvisor</class>
-	</advice>
-	 /AOP -->
-	
-	
-	<!-- Required Privileges -->
-	<privilege>
-		<name>View Common Lab Tests</name>
-		<description>Allows user to access Patient Lab Tests</description>
-	</privilege>
-
-
-	<!-- Required Global Properties 
-	<globalProperty>
-		<property>@MODULE_ID@.someProperty</property>
-		<defaultValue></defaultValue>
-		<description>
-			A description of the global property goes here.  The description
-			should explain enough for administrators to understand the purpose
-			and possible values for the global property.
-		</description>
-	</globalProperty>
-	/Required Global Properties -->
-	
-	<!--
-	<dwr>
-		<allow>
-			<create creator="new" javascript="DWRFormEntryService">
-				<param name="class" value="org.openmrs.module.@MODULE_ID@.web.DWRFormEntryService"/>
-				<include method="enterForm"/>
-			</create>
-		</allow>
-
-		<signatures>
-			<![CDATA[
-			import @MODULE_PACKAGE@.web.DWRFormEntryService;
-			DWRFormEntryService.enterForm(Integer patientId, Integer formId, boolean useEncounter, List<String> fields);
-			]]>
-		</signatures>	
-	</dwr>
-	-->
-	
-	<!-- Servlets -->
-	<!-- Accessed through the url /pageContext()/moduleServlet/<moduleId>/<servlet-name> 
-	<servlet>
-		<servlet-name>formDownload</servlet-name>
-		<servlet-class>@MODULE_PACKAGE@.web.FormDownloadServlet</servlet-class>
-	</servlet>
-	-->
-	<!-- /Servlets -->
-	
-	
-	<!-- Internationalization -->
-	<!-- All message codes should start with @MODULE_ID@.* -->
-	<messages>
-		<lang>en</lang>
-		<file>messages.properties</file>
-	</messages>
-	<messages>
-		<lang>fr</lang>
-		<file>messages.properties</file>
-	</messages>
-	<messages>
-		<lang>es</lang>
-		<file>messages.properties</file>
-	</messages>
-	<!-- /Internationalization -->
-	<mappingFiles>
-		CommonLabTestAttribute.hbm.xml
-	</mappingFiles>
-	
-</module>
-
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Revert this file back and make specific changes only. Currently, the commit shows that whole text was changed.